Riot/Modules/Room/RoomViewController.m

```diff
@@ -4494,7 +4562,7 @@ - (void)refreshMissedDiscussionsCount:(BOOL)force
                 missedDiscussionsBadgeLabel.text = [NSString stringWithFormat:@"%tu", missedCount];
             }
             
-            missedDiscussionsBadgeLabel.textColor = highlightCount ? ThemeService.shared.theme.noticeColor : ThemeService.shared.theme.tintColor;
+            missedDiscussionsDotView.alpha = highlightCount == 0 ? 0 : 1;
```

Comment:
Why using `alpha` instead of `hidden`?

Reply:
I'm using hidden to completely "deactivate" the dot view in `setShowMissedDiscussionsBadge`as hidden property is priority on iOS. I use alpha show/hide the dot view according the highlight count. Alpha is also animatable which may be useful if designer want to animate this dot in case of the user receives an important message.

Reply:
ok. It works for me